### PR TITLE
fix(builder): `output.sourceMap` not work

### PR DESCRIPTION
**Pull request by SoonIter**
## Related Links

@9aoy

## Checklist

- [ ] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/cli/uni-builder/src/shared/parseCommonConfig.ts [hunk 1]
```diff
@@ -326,6 +327,7 @@ export async function parseCommonConfig(
     pluginGlobalVars(globalVars),
     pluginDevtool({
       disableSourceMap,
+      sourceMap,
```
Comment (9aoy) [inline packages/cli/uni-builder/src/shared/parseCommonConfig.ts:330]: If ﻿`output.sourcemap.js` is define, I think we do not need to apply ﻿`pluginDevtool`.

Comment (SoonIter) [reply packages/cli/uni-builder/src/shared/parseCommonConfig.ts:330]: if disableSourceMap and sourceMap are set both, what behavior should we do

Comment (9aoy) [reply packages/cli/uni-builder/src/shared/parseCommonConfig.ts:330]: `output.sourceMap`, because `output.disableSourceMap` is deprecated.

Comment (SoonIter) [reply packages/cli/uni-builder/src/shared/parseCommonConfig.ts:330]: done

**Review by 9aoy — APPROVED** [review]

**Pull request merged**